scrapy_bench/middlewares.py

```diff
@@ -0,0 +1,40 @@
+import os
+
+from scrapy import Request
+
+
+class RandomPayloadMiddleware:
+    def __init__(self, size):
+        """
+        Add a random payload to test network effects
+
+        :param name: size of payload added in bytes
+        :type name: int
+
+        """
+        self.size = size
+
+    @classmethod
+    def from_crawler(cls, crawler):
+        return cls(crawler.settings.getint('SCRAPY_BENCH_RANDOM_PAYLOAD_SIZE'))
+
+    def handle_request(self, smth):
+        if not isinstance(smth, Request):
+            return smth
+
+        request = smth
+
+        if self.size == 0:
+            return request
+
+        request.meta['random_payload'] = os.urandom(self.size)
```

Comment:
@whalebot-helmsman what would be the consequence of settings this meta field, how does this affect the benchmark? It won't be sent over the network, right?

Reply:
Sorry, I didn't check #36 - now I see the context, although still not 100% clearly. Could you please document what is the goal of this setting somewhere, in the README or in the code?

Reply:
done in https://github.com/scrapy/scrapy-bench/pull/37/commits/bae275e47049362a617785d3369b94207396e1c1

Reply:
Thanks @whalebot-helmsman , makes sense - so this meta field would be serialized with the request when using a disk queue, this is why it matters?